dd-sdk-android-ndk/src/test/cpp/test_utils.cpp

```diff
@@ -0,0 +1,17 @@
+#include "test_utils.h"
+
+namespace testutils {
+
+    int count_backtrace_lines(const char *buffer) {
+        int newlines = 0;
+        char *previousAddress = (char *) buffer;
+        for (char c = *buffer; c; c = *++buffer) {
+            if (c == 'n' && *previousAddress == '\\') {
+                newlines++;
+            }
+            previousAddress = (char *) buffer;
```

Comment:
shouldn't it be `previousAddress = c` or something ?

Reply:
Nope...`c` is just a value ...here I am keeping a reference to the previous pointer...it could work also with the value but I prefer pointers.